sdk/storage/azure-storage-blob-changefeed/src/main/java/com/azure/storage/blob/changefeed/BlobChangefeedClientBuilder.java

```diff
@@ -4,10 +4,72 @@
 package com.azure.storage.blob.changefeed;
 
 import com.azure.core.annotation.ServiceClientBuilder;
+import com.azure.core.http.HttpPipeline;
+import com.azure.storage.blob.BlobServiceAsyncClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceVersion;
 
 /**
- * TEMP
+ * This class provides a fluent builder API to help aid the configuration and instantiation of
+ * {@link BlobChangefeedClient BlobChangefeedClients} and {@link BlobChangefeedAsyncClient BlobChangefeedAsyncClients}
+ * when {@link #buildClient() buildClient} and {@link #buildAsyncClient() buildAsyncClient} as called respectively.
  */
 @ServiceClientBuilder(serviceClients = {BlobChangefeedClient.class, BlobChangefeedAsyncClient.class})
-public class BlobChangefeedClientBuilder {
+public final class BlobChangefeedClientBuilder {
+    private final String accountUrl;
+    private final HttpPipeline pipeline;
+    private final BlobServiceVersion version;
+
+    /**
+     * Constructs the {@link BlobChangefeedClientBuilder} using the {@link BlobServiceClient#getAccountUrl() account
+     * URL} and {@link BlobServiceClient#getHttpPipeline() HttpPipeline} properties of the passed {@link
```

Comment:
I think we can just say using the URL (or endpoint) and pipeline of the passed client. I don't think we need to specify that we use the getters.